pkg/podmounter/mppod/creator.go

```diff
@@ -10,9 +10,10 @@ import (
 
 // Labels populated on spawned Mountpoint Pods.
 const (
-	LabelVersion    = "s3.csi.aws.com/mountpoint-version"
-	LabelPodUID     = "s3.csi.aws.com/pod-uid"
-	LabelVolumeName = "s3.csi.aws.com/volume-name"
+	LabelVersion          = "s3.csi.aws.com/mountpoint-version"
```

Comment:
Could you rename this to `LabelMountpointVersion` to make it clear?

Reply:
Done!